### DISCUSSION TITLE
Volume mixer fixed position on start up

### DISCUSSION THREAD

**Discussion by uveeme** [File-New-Project/EarTrumpet#1630] [General]
I’m using the dev version of the app specifically for the "Show full mixer window on startup" feature, and it’s working great! However, I’d like to keep it fixed in place on my secondary screen, but whenever it opens on startup, it appears at the original position instead. Any way to make it remember the position on the secondary screen?

Not sure if it's been discussed before, first time using github and I don't know my way around it yet.

**Comment by riverar** [comment]
Welcome @uveeme! Interesting, I thought I introduced window placement code for that so let me check up on it and get that fixed for you.

**Reply by uveeme** [reply]
Sorry, been away for a few and just got back. After a few times of restarting the PC, the app was then fixed to it's intended position. Thanks for the reply and any help provided!

**Comment by BonnieSoftware** [comment]
by default, Windows opens audio windows to the bottom right, near where you click the volume tray icon that is built into windows, would this not simply things for now by positioning the windows that open to the bottom right as opposed to the top left?
